test/shared/src/main/scala/zio/test/Assertion.scala

```diff
@@ -473,4 +473,52 @@ object Assertion {
         case t: Throwable => assertion(t)
       }
     }
+
+  final def isEmptyString: Assertion[String] = Assertion.assertion(s"isEmpty")() { actual =>
+    actual.isEmpty
+  }
+
+  final def isNonEmptyString: Assertion[String] = Assertion.assertion(s"isNonEmptyString")() { actual =>
```

Comment:
You can make this and the other assertions that don't have any parameters a `val` instead of a `def`.